integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -61,14 +66,27 @@ private static String getInsertWithPartitionSql(
   // Whether supports [CLUSTERED BY col_name3 SORTED BY col_name INTO num_buckets BUCKETS]
   protected abstract boolean supportsSparkSQLClusteredBy();
 
-  // Use a custom database not the original default database because SparkIT couldn't read&write
-  // data to tables in default database. The main reason is default database location is
+  protected abstract boolean supportsPartition();
+
+  // Use a custom database not the original default database because SparkCommonIT couldn't
+  // read&write data to tables in default database. The main reason is default database location is
   // determined by `hive.metastore.warehouse.dir` in hive-site.xml which is local HDFS address
   // not real HDFS address. The location of tables created under default database is like
   // hdfs://localhost:9000/xxx which couldn't read write data from SparkCommonIT. Will use default
   // database after spark connector support Alter database xx set location command.
   @BeforeAll
   void initDefaultDatabase() {
+    // cleanup the metastore_db directory in embedded mode
+    // to avoid the exception about `ERROR XSDB6: Another instance of Derby may have already booted
+    // the database /home/runner/work/gravitino/gravitino/integration-test/metastore_db`
```

Comment:
could you replace `/home/runner/work/gravitino/gravitino/` with `{GRAVITINO_HOME}`? and polish the description to make it more clear?

Reply:
sure

Reply:
updated.